doc/sphinx/source/utils/RTW/add_a_recipe.rst

```diff
@@ -76,47 +72,41 @@ How to add a recipe to the |RTW|
 
     cylc stop recipe_test_workflow/*
 
-#. Run the |RTW|, as detailed in the
-   :doc:`Quick Start Guide <user_guide/quick_start>`; it is expected that the
-   ``compare`` task will fail.
+#. Run the |RTW|, as detailed in the :ref:`quick_start_guide`; it is expected
+   that the ``compare`` task will fail.
 
 #. Update the Known Good Outputs (|KGOs|):
 
    * Recursively copy the recipe output directory (i.e.
-     ``recipe_<recipe>_<date>_<time>``) from the
-     ``${HOME}/cylc-run/recipe_test_workflow/run1/share/cycle/<cycle>``
-     directory to your site-specific KGO directory, as detailed by the
+     ``recipe_<recipe>_<date>_<time>/``) from the
+     ``${HOME}/cylc-run/recipe_test_workflow/run1/share/cycle/<cycle>/``
```

Comment:
```suggestion
     ``${HOME}/cylc-run/recipe_test_workflow/runN/share/cycle/<cycle>/``
```
Use `runN`, which links to latest run, just in case the user makes multiple attempts and doesn't `cylc clean`?

Reply:
Thanks @alistairsellar! Done in ea4257713 😊